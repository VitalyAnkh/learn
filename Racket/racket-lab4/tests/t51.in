(intersect '(to be) (union '(or not to be) '(that is the question)))
(union '(to be) (difference '(or not to be) '(that is the question)))
(difference '(to be) (intersect '(or not to be) '(that is the question)))
(union '(to be) (intersect '(or not to be) (union '(that is the question) '())))
(difference '(to be) (difference '(or not to be) '(that is the question)))
(intersect '(to be) (difference '(or not to be) (union '(union '(that) '()) '(is the question))))
(union '(to be) (union (union '(or not) '(to be)) (union (difference '(that) '(is)) '(the question))))
(intersect '(to be) (union '(or not to be) (difference '(that is) '(the question))))
