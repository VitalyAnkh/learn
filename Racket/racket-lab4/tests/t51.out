'(to be)
'(or not to be)
'(to be)
'(be to)
'()
'(to be)
'(be to or not that the question)
'(to be)
